core/src/main/java/com/amplifyframework/util/Wrap.java

```diff
@@ -24,6 +24,20 @@
 public final class Wrap {
     private Wrap() {}
 
+    /**
+     * Returns original string wrapped with in backticks.
+     * @param original Original string to modify.
+     * @return Original string wrapped with backtick.
+     *         If original string is null or empty, it just returns the original.
```

Comment:
this description is not true for empty strings right? you can use [`Empty#check(String)`](https://github.com/aws-amplify/amplify-android/blob/b0980c7e4f8e3cb0729c8136a4e186fd628433f6/core/src/main/java/com/amplifyframework/util/Empty.java#L58-L60) utility method to return the original string if true.

Reply:
Good catch. Fixed in next commit